### PR TITLE
fix(ActivityCenter): Hide pinned messages

### PR DESCRIPTION
**Pull request by borismelnik**
Closes: #5205

### What does the PR do

Hide pinned messages

### Affected areas

Activity Center

### CODE REVIEW

**Review by jrainville — COMMENTED** [review]
There is something messed up deeper in the activity center workflow if a pinned message was allowed to be displayed.
I would be good to investigate what that message is and why it got there. Can you check the `type` of notification that was? And if it really was something that shouldn't have been put there, we should fix it in the service or module instead of in the QML ,in my opinion

**Comment by borismelnik** [comment]
> There is something messed up deeper in the activity center workflow if a pinned message was allowed to be displayed.
> I would be good to investigate what that message is and why it got there. Can you check the `type` of notification that was? And if it really was something that shouldn't have been put there, we should fix it in the service or module instead of in the QML ,in my opinion

Agreed, but I checked it. Notification type is OneToOne and I didn't find any other way(

**Review by saledjenic — CHANGES_REQUESTED** [review]
@borismelnik it's very interesting that "Pinned By" label is displayed at all since it's by default set to "false", and I briefly checked the flow there, actually where/when we set `pinned` property to `true` and all looked good, but I've found something very weird in `message_item_qobject.nim`, check [this](https://github.com/status-im/status-desktop/blob/master/src/app/modules/shared_models/message_item_qobject.nim#L113-L115). Not sure why line `114` is like that? It should be:
`QtProperty[bool] pinned:` instead of `QtProperty[int] bool:`

Please updated that and check what's happening (somehow I expect that's the origin of the issue, didn't test). 

Also within this PR, please change other props there as well which should be bool, like in lines `118` and `122`.

**Comment by jrainville** [comment]
> but I've found something very weird in message_item_qobject.nim, check [this](https://github.com/status-im/status-desktop/blob/master/src/app/modules/shared_models/message_item_qobject.nim#L113-L115). Not sure why line 114 is like that? It should be:
QtProperty[bool] pinned: instead of QtProperty[int] bool:

Good find Sale. I have no idea why it got like that.  assume bad copy paste. We for sure need to fix those (the ones below as well)

File: src/app/modules/shared_models/message_item_qobject.nim [hunk 1]
```diff
@@ -111,15 +111,15 @@ QtObject:
   #   read = reactionIds
 
   proc pinned*(self: MessageItem): bool {.slot.} = result = ?.self.messageItem.pinned
-  QtProperty[int] bool:
+  QtProperty[bool] bool:
```
Comment (saledjenic) [inline src/app/modules/shared_models/message_item_qobject.nim:114]: @borismelnik but man, why did you leave `bool` as name here?

In my previous comment I said you should update to `QtProperty[bool] pinned:` instead `QtProperty[int] bool:`, also you should add name which is used on the qml side for other 2 cases.

**Review by saledjenic — CHANGES_REQUESTED** [review]

File: src/app/modules/shared_models/message_item_qobject.nim (hunk 1 above)
Comment (borismelnik) [reply src/app/modules/shared_models/message_item_qobject.nim:114]: oh :D

**Review by saledjenic — APPROVED** [review]

**Pull request merged**